hq/app/aws/ec2/EC2.scala

```diff
@@ -8,7 +8,7 @@ import cats.instances.set._
 import cats.syntax.semigroup._
 import com.amazonaws.auth.AWSCredentialsProviderChain
 import com.amazonaws.regions.Regions
-import com.amazonaws.services.ec2.model.{DescribeNetworkInterfacesRequest, DescribeNetworkInterfacesResult, Filter, NetworkInterface}
+import com.amazonaws.services.ec2.model.{Instance => _, _}
```

Comment:
This `Instance => _` import is a bad sign. Might mean we need to rethink our own model name because this clash is something that could easily cause lots of confusion.